clients/spark/core/src/main/scala/io/treeverse/clients/ApiClient.scala

```diff
@@ -57,7 +57,8 @@ class ApiClient(
     apiUrl: String,
     accessKey: String,
     secretKey: String,
-    connectionTimeout: String = ""
+    connectionTimeout: String = "",
```

Comment:
Can these be ints? I prefer for the app to convert strings to numbers it makes for reusable lib code _and_ makes it easy for the app to give good error messages.

Reply:
Also, durations should either be typed (like time.Duration in Go or their variables should include a unit in the name, say connectionTimeoutSecs.

Reply:
opened https://github.com/treeverse/lakeFS/issues/3989 to follow up on these great comments